src/components/slider/slider.scss

```diff
@@ -0,0 +1,141 @@
+@import 'default-theme';
+
+// This refers to the thickness of the slider. On a horizontal slider this is the height, on a
+// vertical slider this is the width.
+$slider-thickness: 20px !default;
```

Comment:
Can we prefix these Sass variables? That's what we're doing elsewhere.  `$slider` -> `$md-slider`